libsolidity/analysis/FunctionCallGraph.cpp

```diff
@@ -199,6 +199,7 @@ bool FunctionCallGraphBuilder::visit(MemberAccess const& _memberAccess)
 		solAssert(*_memberAccess.annotation().requiredLookup == VirtualLookup::Static, "");
 
 	functionReferenced(*functionDef, _memberAccess.annotation().calledDirectly);
+
```

Comment:
Is this a leftover or on purpose?

Reply:
Well.. kind of both? visually I think it's better, but it only is in the PR because I touched & reverted things in that file...